sat/cli/bootsys/mgmt_power.py

```diff
@@ -260,6 +260,86 @@ def finish_shutdown(hosts, username, password, ncn_shutdown_timeout, ipmi_timeou
             sys.exit(0)
 
 
+def get_disk_entries(ssh_client, ncn):
+    """
+    Retrieve the current boot order from a node via SSH and identify disk entries.
+
+    Args:
+        ssh_client (paramiko.SSHClient): The SSH client connected to the node.
+        ncn (str): The node associated with the ssh_client.
+
+    Returns:
+        str: The boot number of the first disk entry found.
+
+    Raises:
+        SystemExit: If the command fails, an error is logged and the function exits.
+    """
+    command = "efibootmgr"
+    _, stdout, stderr = ssh_client.exec_command(command)
+    error = stderr.read().decode()
+    if error:
+        LOGGER.error(f'Failed to get current boot order for {ncn}: {error}')
+        raise SystemExit(1)
+
+    boot_order_output = stdout.read().decode()
+    lines = boot_order_output.split('\n')
+    for line in lines:
+        if line.startswith("Boot"):
+            parts = line.split()
+            boot_num = parts[0][4:8]
+            boot_desc = " ".join(parts[1:])
+            if "UEFI OS" in boot_desc:
+                return boot_num
+    return None
+
+
+def set_next_boot_device_to_disk(ssh_client, ncns):
+    """
+    Set the next boot device to a disk for a list of nodes via SSH.
+
+    This function connects to each node via SSH, retrieves the current boot order, identifies the first available disk,
+    and sets the next boot device to the specified disk.
+
+    Args:
+        ssh_client (paramiko.SSHClient): An active SSH client used to connect to the nodes.
+        ncns (list): A list of node names (NCNs) to configure.
+
+    Raises:
+        paramiko.SSHException: If there is an error connecting to any node via SSH.
+        ValueError: If there are no disk boot entries found for a node.
+        RuntimeError: If there are issues setting the next boot device.
+    """
+    for ncn in ncns:
+        try:
+            ssh_client.connect(ncn)
+        except (SSHException, socket.error) as err:
+            LOGGER.warning('Unable to connect to node "%s": %s', ncn, err)
+            continue
+
+        try:
+            disk_boot_num = get_disk_entries(ssh_client, ncn)
+            if not disk_boot_num:
+                raise ValueError(f'No disk boot entries found for {ncn}')
+
+            LOGGER.info(f'New Boot Order for {ncn}: {disk_boot_num}')
+
+            # Set the next boot device
+            set_boot_next = f"efibootmgr -n {disk_boot_num}"
+            _, stdout, stderr = ssh_client.exec_command(set_boot_next)
+            error = stderr.read().decode()
+            if error:
+                raise RuntimeError(f'Failed to set next boot device for {ncn}: {error}')
+
+            LOGGER.info(f'Successfully set next boot device to disk for {ncn}')
```

Comment:
Perhaps include the boot option number here as well, and then remove the info log on line 324.

```suggestion
            LOGGER.info(f'Successfully set next boot device to disk (Boot{disk_boot_num}) for {ncn}')
```

Reply:
Addressed